src/test/mochitest/head.js

```diff
@@ -572,8 +586,7 @@ async function stepOut(dbg) {
  * @return {Promise}
  * @static
  */
-function resume(dbg) {
-  info("Resuming");
+async function resume(dbg) {
```

Comment:
does this need the async keyword? if the dbg.actions.resume() is a promise, I think you can `await` it